src/defaults/fileIcons.ts

```diff
@@ -1210,6 +1210,10 @@ const fileIcons: Record<string, {
       '.whitesource',
     ],
   },
+  'json-schema': {
+    fileNames: ['schema.json'],
```

Comment:
im a bit unsure if using such a generic file name is a good idea
while the extension `.schema.json` does make sense, the filename could reference something different from a json schema